classes/ActionScheduler_Versions.php

```diff
@@ -122,65 +128,24 @@ public static function initialize_latest_version() {
 	 *         'name' => 'Name',
 	 *     ]
 	 *
+	 * @deprecated 3.9.1 Use ActionScheduler_SystemInformation::active_source().
```

Comment:
```suggestion
	 * @deprecated 3.9.2 Use ActionScheduler_SystemInformation::active_source().
```

Building on the discussion in [this thread](https://github.com/woocommerce/action-scheduler/pull/1237/files#r1935667047), this version number should be bumped to 3.9.2.